src/app/CommandResponseSender.h

```diff
@@ -123,6 +123,26 @@ class CommandResponseSender : public Messaging::ExchangeDelegate
         mChunks.AddToEnd(std::move(aPacket));
     }
 
+    /**
+     * @brief Sets final non-path specific StatusResponse error after sending all InvokeResponseMessages
```

Comment:
Is this after _sending_ all the InvokeResponseMessages?  Or after _queueing_ them?  This is not making sense to me as documented.